scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/StaxJobFactory.java

```diff
@@ -867,7 +867,7 @@ private void handleCancelJobOnErrorAttribute(CommonAttribute commonPropertiesHol
         LinkedHashMap<String, TaskVariable> updatedVariablesMap = new LinkedHashMap<>(variablesMap);
```

Comment:
(non mandatory) i could have been great to rename all non relevant map name, by using jobVariables, jobSubmissionVariables, resolvedTaskVariables,..